spec/rspec/core/bisect/runner_spec.rb

```diff
@@ -224,8 +245,35 @@ def repro_command_from(ids)
       end
 
       it "runs the suite with the original CLI options" do
-        runner.original_results
-        expect(Open3).to have_received(open3_method).with(a_string_including("--seed 1234"))
+        expect {
+          runner.original_results
+        }.to invoke_command_with_env(a_string_including("--seed 1234"), {})
+      end
+
+      context 'when --bisect is present in SPEC_OPTS' do
+        it "runs the suite with --bisect removed from the environment" do
```

Comment:
you can test the effect of removing `--bisect` from the SPEC_OPTS separately from modifying the SPEC_OPTS when invoking the command , if you wish, using something like:

```ruby
with_env_var 'SPEC_OPTS' => '--bisect --fail-fast' do
  expect(@runner.spec_opts_without_bisect).to eq('--fail-fast')
end
# and then
with_env_vars 'RSPEC_TEMP' => 'original' do
  actual = @runner.run_command "ruby -e 'puts ENV['RSPEC_TEMP']"
  expect(actual).to eq('original')
end
```